tooling/api/src/dialog.ts

```diff
@@ -116,38 +116,22 @@ interface MessageDialogOptions {
  *     extensions: ['png', 'jpeg']
  *   }]
  * });
- * if (Array.isArray(selected)) {
- *   // user selected multiple files
- * } else if (selected === null) {
- *   // user cancelled the selection
- * } else {
- *   // user selected a single file
- * }
- * ```
- *
- * @example
- * ```typescript
- * import { open } from '@tauri-apps/api/dialog';
- * import { appDir } from '@tauri-apps/api/path';
- * // Open a selection dialog for directories
- * const selected = await open({
- *   directory: true,
- *   multiple: true,
- *   defaultPath: await appDir(),
- * });
- * if (Array.isArray(selected)) {
- *   // user selected multiple directories
- * } else if (selected === null) {
- *   // user cancelled the selection
- * } else {
- *   // user selected a single directory
- * }
  * ```
+ * Note that the `open` function returns a conditional type depending on the `multiple` option;
+ * - Not specified -> `Promise<string | null>`
+ * - false -> `Promise<string | null>`
+ * - true -> `Promise<string[] | null>`
```

Comment:
```suggestion
 * - false (default) -> `Promise<string | null>`
 * - true -> `Promise<string[] | null>`
```
What do you think about combining these 2 lines like this? Or is that not clear enough?

Reply:
LGTM!